### ISSUE TITLE
CartesianPoseController should output directly to joint torques  (ros ticket #3456)

### ISSUE THREAD

**Issue by ahendrix** [PR2/pr2_controllers#245]
Currently the cartesian pose controller outputs to twists.  Instead, it should output directly to joint torques.

trac data:
- Owner: **sglaser**
- Reporter: **sglaser**
- Reported at: **Thu Jan  7 18:03:05 2010**
- URL: https://code.ros.org/trac/wg-ros-pkg/ticket/3456

**Comment by ahendrix** [comment]
[wim] I assume that pose controller will use J^T and not J^-1?

**Comment by ahendrix** [comment]
[sglaser] Yeah, it's a JT.

r28718

**Comment by ahendrix** [comment]
[wim] Could you check on the door test? I think this patch broke the test. Thanks!

controllers config in milestone2_common/config/controllers.xml

**Comment by ahendrix** [comment]
[sglaser] M2 should be fixed in r28755

**Comment by ahendrix** [comment]
[wim] We have a problem here... I got the milestone tests to run again, but the robot gets stuck in the door because the elbow sticks out too much. So the constraint we removed from the wrench controller was really needed by this test. We really should keep this test up and running, so I guess we can either revert the change in the wrench controller, or copy the old code of the wrench controller in experimental_controllers.
